near/omni-bridge/src/storage.rs

```diff
@@ -170,6 +176,13 @@ impl Contract {
         self.accounts_balances.get(account_id)
     }
 
+    pub fn has_storage_balance(&self, account_id: &AccountId, balance: NearToken) -> bool {
```

Comment:
Do we really need this a public method?

Reply:
Impossible to use it as private from the main file

Reply:
Let's use `pub(crate)`

Reply:
8ecbc4ba99a780e2fccf1d1b0c8c260d3997fcfd